UDV-Core/src/Utils/3DTiles/3DTilesUtils.md

```diff
@@ -0,0 +1,207 @@
+# Helper functions for 3DTiles
```

Comment:
Add a link to this doc in 3DTilesUtils.js  and 3DTilesBuildingUtils.js please

Reply:
Done :)